azext_iot/operations/dps.py

```diff
@@ -193,6 +194,8 @@ def iot_dps_device_enrollment_group_create(client,
                                            resource_group_name,
                                            certificate_path=None,
                                            secondary_certificate_path=None,
+                                           certificate_name=None,
+                                           secondary_certificate_name=None,
                                            iot_hub_host_name=None,
                                            initial_twin_tags=None,
```

Comment:
For processing initial_twin_tags and initial_twin_properties (like in _get_initial_twin)...use azext_iot.common.utility.shell_safe_json_parse this allows you to support json _or_ python dictionary notation.